js/index.js

```diff
@@ -1 +1,190 @@
 // Your code goes here
+// * [ ] Using your [index.js file](js/index.js), create 10 [unique event listeners](https://developer.mozilla.org/en-US/docs/Web/Events). using your creativity to make the Fun Bus site more interactive.  Here are some unique events you could try to use:
+// 	* [ ] `mouseover`
+// 	* [ ] `keydown`
+// 	* [ ] `wheel`
+// 	* [ ] `drag / drop`
+// 	* [ ] `load`
+// 	* [ ] `focus`
+// 	* [ ] `resize`
+// 	* [ ] `scroll`
+// 	* [ ] `select`
+// 	* [ ] `dblclick`
+
+const cl = console.log;
+const navButtons = document.querySelectorAll(".nav-link");
+const lastPic = document.querySelector(".content-destination img");
+const destinations = document.querySelectorAll(".destination .btn");
+const destinationDivs = document.querySelectorAll(".destination");
+const centerTitleTop = document.querySelector(".content-section h2");
+const centerTitleBottom = document.querySelector(".inverse-content h2");
+const topPic = document.querySelector(".intro img");
+const topTitle = document.querySelector(".logo-heading");
+
+// 1 & 2 navbar borders pop in with mouseover and return to normal with mouseleave
+const borderBoxOn = function() {
+  this.style.border = "1px solid black";
```

Comment:
Great job structuring this! By creating the functions separately you are able to reuse them.